### PR TITLE
Create MenuItemReview Database

### LINKED ISSUE #15: Create database table for MenuItemReview
Create database table for MenuItemReview

# Acceptance Criteria:

- [x] There is an `@Entity` class called MenuItemReview.java
- [x] There is a `@Repository` class called MenuItemReviewRepository.java
- [x] When you start up the repo on localhost, you can see the table
      using the H2 console (see the file `docs/h2-database.md` for 
      instructions.)
- [ ] When you deploy the app to Heroku, you can see the table
      using the psql command (See `Accessing Database Console` in the
      main `README.md` file for instructions.)

### PR DESCRIPTION
**Pull request by wesleykam**
In this PR, I created the MenuItemReview Database with the @Entity and @Repository classes.

closes #15

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/example/entities/MenuItemReview.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "menuitemreview")
```
Comment (ethwu) [inline src/main/java/edu/ucsb/cs156/example/entities/MenuItemReview.java]: I think this should be `"menuitemreviews"`.

**Review by ethwu — CHANGES_REQUESTED** [review]
Need to update entity name but otherwise lgtm!

**Comment by wesleykam** [comment]
Fixed issue waiting for reapproval

**Review by ethwu — APPROVED** [review]
LGTM1

**Pull request merged**